backbone/src/main/java/org/researchstack/backbone/model/survey/NavigationStep.java

```diff
@@ -0,0 +1,24 @@
+package org.researchstack.backbone.model.survey;
+
+import org.researchstack.backbone.result.StepResult;
+import org.researchstack.backbone.result.TaskResult;
+import org.researchstack.backbone.step.QuestionStep;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 12/31/16.
+ */
+
+public interface NavigationStep {
```

Comment:
This is an unusual interface to have getters/setter like this.  I made it to keep the code similar to the iOS Swift version.  In that language you can member variables in interfaces.